### PR TITLE
net-im/telegram-desktop: Fix building with KWayland 5.93

### PR DESCRIPTION
**Pull request by mid-kid**
@gyakovlev

Not revbumping since this doesn't affect runtime, only rebuilds.

### CODE REVIEW

File: net-im/telegram-desktop/files/tdesktop-3.6.1-fix-kwayland-5.93.patch [hunk 1]
```diff
@@ -0,0 +1,53 @@
+Fix compatibility with kwayland-5.93.0
+
+An include path that was previously being relied on isn't present anymore with the new release
```
Comment (thesamesam) [inline net-im/telegram-desktop/files/tdesktop-3.6.1-fix-kwayland-5.93.patch:3]: Please include a link to the Gentoo bug and upstream bug/PR at the top of the patch (do this for all patches in future too)

Comment (mid-kid) [reply net-im/telegram-desktop/files/tdesktop-3.6.1-fix-kwayland-5.93.patch]: Fixed

**Comment by mid-kid** [comment]
@thesamesam requesting re-review and merge this solves a fairly common build error. (I've never had this many CCs in a bug report)

File: net-im/telegram-desktop/files/tdesktop-3.6.1-fix-kwayland-5.93.patch [hunk 2]
```diff
@@ -0,0 +1,55 @@
+Fix compatibility with kwayland-5.93.0
```
Comment (thesamesam) [inline net-im/telegram-desktop/files/tdesktop-3.6.1-fix-kwayland-5.93.patch:1]: Does this work with < 5.93.0?

Comment (mid-kid) [reply net-im/telegram-desktop/files/tdesktop-3.6.1-fix-kwayland-5.93.patch:1]: yes

Comment (thesamesam) [reply net-im/telegram-desktop/files/tdesktop-3.6.1-fix-kwayland-5.93.patch:1]: great, thanks!

**Pull request closed without merging**